Assets/Thirdweb/Core/Scripts/TransactionManager.cs

```diff
@@ -118,7 +118,7 @@ public static async Task<TransactionReceipt> ThirdwebWriteRawResult<TWFunction>(
 
                 var postData = new RelayerRequest(request, signature, forwarderAddress);
 
-                using UnityWebRequest req = UnityWebRequest.Post(relayerUrl, "");
+                using UnityWebRequest req = UnityWebRequest.PostWwwForm(relayerUrl, "");
```

Comment:
Please keep this as post to be backwards compatible with 2021

Reply:
Oh, I see. I'll do it right away